### DISCUSSION TITLE
Are Software Hacking Ilegal In github

### DISCUSSION THREAD

**Discussion by BangKuntet** [community/community#21337] [Other Feature Feedback, Questions, & Ideas] [answered]
do you make source code repo about illegal software hacking forbiden on github

**Comment by Zusyaku** [comment, marked as answer]
no github is a place to make open source illegal programs or not its because user

**Reply by jfullstackdev** [reply]
what do you mean by this ?

**Comment by shakexe** [comment]
I think you can make hacking links and api her 
Since I received a link from github and hacked my snapchat 🤷🏻‍♂️
you can check the ink but dont click it 
[link redacted for safety concerns].

**Reply by queenofcorgis** [reply]
Hi @shakexe 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
